synapsebridgehelpers/tableHelpers.py

```diff
@@ -43,11 +43,11 @@ def find_tables_with_data(syn, tables, healthCodes):
 def safe_query(query_str, syn, continueOnMissingColumn):
     try:
         return syn.tableQuery(query_str)
-    except sc.exceptions.SynapseHTTPError as e:
+    except sc.coreexceptions.SynapseHTTPError as e:
```

Comment:
Isn't this missing a  . between core and exceptions

Reply:
Fixed. Clearly the test coverage is not 100% 🤔